examples.PubsubliteToGcsIT: testPubsubliteToGcs failed

Note: #8657 was also for this test, but it was closed more than 10 days ago. So, I didn't mark it flaky.

----

commit: 9b0400ac5290e2af2ad13f3e75c2aed25c7764ce
buildURL: [Build Status](https://source.cloud.google.com/results/invocations/5bf08ec1-2409-4291-86e8-be10522417fd), [Sponge](http://sponge2/5bf08ec1-2409-4291-86e8-be10522417fd)
status: failed
<details><summary>Test output</summary><br><pre>java.util.concurrent.ExecutionException: com.google.api.gax.rpc.NotFoundException: io.grpc.StatusRuntimeException: NOT_FOUND: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC does not exist.
	at pubsublite.streaming@1.0.0-SNAPSHOT/examples.PubsubliteToGcsIT.tearDown(PubsubliteToGcsIT.java:152)
Caused by: com.google.api.gax.rpc.NotFoundException: io.grpc.StatusRuntimeException: NOT_FOUND: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC does not exist.
Caused by: io.grpc.StatusRuntimeException: NOT_FOUND: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC does not exist.
</pre></details>

# Java 17 - Periodic
Looks like something is getting cleaned up before it can be used.  Perhaps the resource janitor?
@kweinmeister FYI
```

#
# pubsublite/streaming-analytics
#

[0.001s][warning][os,container] Duplicate cpuset controllers detected. Picking /sys/fs/cgroup/cpuset, skipping /tmpfs/docker/overlay2/99e20965e286f46873a0a797da9ddd92e48c24f9bb276f5a344e60707c13bc62/merged/sys/fs/cgroup/cpuset.
[ERROR] Tests run: 5, Failures: 1, Errors: 4, Skipped: 0, Time elapsed: 395.685 s <<< FAILURE! - in examples.PubsubliteToGcsIT
[ERROR] examples.PubsubliteToGcsIT.testPubsubliteToGcs  Time elapsed: 394.342 s  <<< FAILURE!
java.lang.AssertionError
	at pubsublite.streaming@1.0.0-SNAPSHOT/examples.PubsubliteToGcsIT.testPubsubliteToGcs(PubsubliteToGcsIT.java:246)

[ERROR] examples.PubsubliteToGcsIT.testPubsubliteToGcs  Time elapsed: 394.343 s  <<< ERROR!
java.util.concurrent.ExecutionException: com.google.api.gax.rpc.FailedPreconditionException: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC was deleted at 2023-10-17T03:41:41.731172-07:00 which is less than 1h ago, it cannot yet be recreated.
	at pubsublite.streaming@1.0.0-SNAPSHOT/examples.PubsubliteToGcsIT.setUp(PubsubliteToGcsIT.java:141)
Caused by: com.google.api.gax.rpc.FailedPreconditionException: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC was deleted at 2023-10-17T03:41:41.731172-07:00 which is less than 1h ago, it cannot yet be recreated.
Caused by: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC was deleted at 2023-10-17T03:41:41.731172-07:00 which is less than 1h ago, it cannot yet be recreated.

[ERROR] examples.PubsubliteToGcsIT.testPubsubliteToGcs  Time elapsed: 394.343 s  <<< ERROR!
java.util.concurrent.ExecutionException: com.google.api.gax.rpc.NotFoundException: io.grpc.StatusRuntimeException: NOT_FOUND: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC does not exist.
	at pubsublite.streaming@1.0.0-SNAPSHOT/examples.PubsubliteToGcsIT.tearDown(PubsubliteToGcsIT.java:152)
Caused by: com.google.api.gax.rpc.NotFoundException: io.grpc.StatusRuntimeException: NOT_FOUND: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC does not exist.
Caused by: io.grpc.StatusRuntimeException: NOT_FOUND: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC does not exist.

[ERROR] examples.PubsubliteToGcsIT.testPubsubliteToGcs  Time elapsed: 394.344 s  <<< ERROR!
java.util.concurrent.ExecutionException: com.google.api.gax.rpc.FailedPreconditionException: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC was deleted at 2023-10-17T03:41:41.731172-07:00 which is less than 1h ago, it cannot yet be recreated.
	at pubsublite.streaming@1.0.0-SNAPSHOT/examples.PubsubliteToGcsIT.setUp(PubsubliteToGcsIT.java:141)
Caused by: com.google.api.gax.rpc.FailedPreconditionException: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC was deleted at 2023-10-17T03:41:41.731172-07:00 which is less than 1h ago, it cannot yet be recreated.
Caused by: io.grpc.StatusRuntimeException: FAILED_PRECONDITION: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC was deleted at 2023-10-17T03:41:41.731172-07:00 which is less than 1h ago, it cannot yet be recreated.

[ERROR] examples.PubsubliteToGcsIT.testPubsubliteToGcs  Time elapsed: 394.349 s  <<< ERROR!
java.util.concurrent.ExecutionException: com.google.api.gax.rpc.NotFoundException: io.grpc.StatusRuntimeException: NOT_FOUND: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC does not exist.
	at pubsublite.streaming@1.0.0-SNAPSHOT/examples.PubsubliteToGcsIT.tearDown(PubsubliteToGcsIT.java:152)
Caused by: com.google.api.gax.rpc.NotFoundException: io.grpc.StatusRuntimeException: NOT_FOUND: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC does not exist.
Caused by: io.grpc.StatusRuntimeException: NOT_FOUND: Resource projects/779844219229/locations/us-central1-b/topics/pubsublite-streaming-analytics-topic-211a8c of type TOPIC does not exist.

[ERROR] Errors:
[ERROR] examples.PubsubliteToGcsIT.testPubsubliteToGcs
[ERROR]   Run 1: PubsubliteToGcsIT.testPubsubliteToGcs:246
[ERROR]   Run 2: PubsubliteToGcsIT.setUp:141 » Execution com.google.api.gax.rpc.FailedPrecondit...
[ERROR]   Run 3: PubsubliteToGcsIT.tearDown:152 » Execution com.google.api.gax.rpc.NotFoundExce...
[ERROR]   Run 4: PubsubliteToGcsIT.setUp:141 » Execution com.google.api.gax.rpc.FailedPrecondit...
[ERROR]   Run 5: PubsubliteToGcsIT.tearDown:152 » Execution com.google.api.gax.rpc.NotFoundExce...
[ERROR] Tests run: 1, Failures: 0, Errors: 1, Skipped: 0
 ```

--- a/pubsublite/streaming-analytics/src/test/java/examples/PubsubliteToGcsIT.java
+++ b/pubsublite/streaming-analytics/src/test/java/examples/PubsubliteToGcsIT.java
@@ -142,8 +142,7 @@ public class PubsubliteToGcsIT {
       System.out.println(responseTopic.getAllFields() + " created successfully.");
       Subscription response = adminClient.createSubscription(subscription).get();
       System.out.println(response.getAllFields() + " created successfully.");
-    }
-    catch (ExecutionException e) {
+    } catch (ExecutionException e) {
       e.printStackTrace();
     }
   }
@@ -156,8 +155,7 @@ public class PubsubliteToGcsIT {
       System.out.println("Deleted topic: " + topicPath);
       adminClient.deleteSubscription(subscriptionPath).get();
       System.out.println("Deleted subscription: " + subscriptionPath);
-    }
-    catch (ExecutionException e) {
+    } catch (ExecutionException e) {
       e.printStackTrace();
     }
 

--- a/pubsublite/streaming-analytics/src/test/java/examples/PubsubliteToGcsIT.java
+++ b/pubsublite/streaming-analytics/src/test/java/examples/PubsubliteToGcsIT.java
@@ -143,6 +143,9 @@ public class PubsubliteToGcsIT {
       Subscription response = adminClient.createSubscription(subscription).get();
       System.out.println(response.getAllFields() + " created successfully.");
     }
+    catch (ExecutionException e) {
+      e.printStackTrace();
+    }
   }
 
   @After
@@ -154,6 +157,10 @@ public class PubsubliteToGcsIT {
       adminClient.deleteSubscription(subscriptionPath).get();
       System.out.println("Deleted subscription: " + subscriptionPath);
     }
+    catch (ExecutionException e) {
+      e.printStackTrace();
+    }
+
 
     // Delete the output files.
     Page<Blob> blobs = storage.list(bucketName, Storage.BlobListOption.prefix(directoryPrefix));

